kudos.py

```diff
@@ -9,14 +9,14 @@
 #}
 
 def load_config():
-	cfg = configparser.ConfigParser()
-	cfg.read('config.ini')
+        with open("config.yml", 'r') as ymlfile:
```

Comment:
You don't need 'r', as it's the default.  Also, let's try to use a consistent style of quotes (single quotes).